xml/System.Text.Unicode/Utf8.xml

```diff
@@ -41,15 +41,29 @@
         <Parameter Name="isFinalBlock" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="charsRead">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <param name="replaceInvalidSequences">To be added.</param>
-        <param name="isFinalBlock">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">A UTF-16 encoded read-only character span.</param>
+        <param name="destination">A UTF-8 encoded byte span.</param>
+        <param name="charsRead">When the method returns, the number of characters read from <paramref name="source" />.</param>
+        <param name="bytesWritten">When the method returns, the number of characters written to <paramref name="destination" />.</param>
+        <param name="replaceInvalidSequences"><see langword="true" /> to replace invalid UTF-16 sequences in <paramref name="source" /> with U+FFFD; <see langword="false" /> to return <see cref="F:System.Buffers.OperationStatus.InvalidData" /> if invalid characters are found in <paramref name="source" />.</param>
+        <param name="isFinalBlock"><see langword="true" /> if the method should not return <see cref="F:System.Buffers.OperationStatus.InvalidData" />; otherwise, <see langword="false" />.</param>
```

Comment:
@rpetrusha - this documentation is inaccurate. I have submitted a fix for it.

https://github.com/dotnet/coreclr/blob/d4ab484be11b54eb92204cd084638a03a8f9c441/src/System.Private.CoreLib/shared/System/Text/Unicode/Utf8.cs#L30-L31

See https://github.com/dotnet/dotnet-api-docs/pull/3208